### ISSUE TITLE
IsCountable validator is not registered in ValidatorPluginManager

### ISSUE THREAD

**Issue by gennadiylitvinyuk** [laminas/laminas-validator#110]
### Bug Report

|    Q        |   A
|------------ | ------
| Version(s)  | 2.12

#### Current behavior

The `IsCountable` validator is not registered in `ValidatorPluginManager` and can not be created with this Plugin-Manager for example with InputFilter.

#### How to reproduce

```
use Laminas\InputFilter\Factory;

$factory = new Factory();
$inputFilter = $factory->createInputFilter([
    'password' => [
        'name'       => 'c',
        'required'   => true,
        'validators' => [
            [
                'name' => 'not_empty',
            ],
            [
                'name' => 'IsCountable', // or is_countable
            ],
        ],
    ],
]);
```

#### Expected behavior

Input Filter could be created

**Comment by rieschl** [comment]
If I try your snippet, the first error I get is that `not_empty` validator was not found. It should be `notempty`, `notEmpty`, `NotEmpty` or `\Laminas\Validator\NotEmpty::class`.

Also, if you use the FQCN of the `IsCountable` validator (`\Laminas\Validator\IsCountable::class`) and the `\Laminas\ServiceManager\AbstractPluginManager::$autoAddInvokableClass` setting is set to `true` (default), the PluginManager will try to create the Validator with the `\Laminas\ServiceManager\Factory\InvokableFactory`, which should be okay for the `IsCountable` validator.

TL;DR: use FQCNs as your validator `name`s.

**Comment by gennadiylitvinyuk** [comment]
Hello @rieschl, @boesing 

In this case we have another Issue, because I've posted an adapted example from the official docs: https://docs.laminas.dev/laminas-inputfilter/intro/ 

At some point in the past `'not_empty'` was removed from the alias list...

As for the most other validators there is an alias, I thought it is an error, if for this one the alias is missing ....

**Comment by froschdesign** [comment]
> …use FQCNs as your validator `name`s.

While this is the right way to go, the other variants should be added because of the consistency.

Like for `Laminas\Validator\IsInstanceOf` and others:

https://github.com/laminas/laminas-validator/blob/270380e87904f5a1a1fba3059989d4ca157e16a9/src/ValidatorPluginManager.php#L184-L186

**Comment by froschdesign** [comment]
@gennadiylitvinyuk 
> In this case we have another Issue, because I've posted an adapted example from the official docs: https://docs.laminas.dev/laminas-inputfilter/intro/

Good catch! Please create a separated issue report on laminas-inputfilter.

Thanks in advance! 👍
